src/types/Person.ts

```diff
@@ -3,9 +3,11 @@ export interface Person {
   sex: string;
   born: number;
   died: number;
-  fatherName: string | null;
-  motherName: string | null;
+  fatherName?: string;
+  motherName?: string;
   slug: string;
   mother?: Person;
   father?: Person;
+  century?: number;
+  [key: string]: string | number | Person | undefined;
```

Comment:
The index signature `[key: string]: string | number | Person | undefined;` allows any additional properties on the `Person` object, which might lead to unintended usage. Consider removing it if you want to enforce strict typing, or ensure it's necessary for your use case.